### PR TITLE
Tech 1048/cacluate cost elastic ip

### PR DESCRIPTION
**Pull request by jakepage91**
Hey there! 

I've added a cost calculation part for Elastic IPs here. But, the AWS Go SDK doesn't let me find the exact creation time of the Elastic IP. So, I've come up with a workaround that calculates the cost of unassociated Elastic IPs from the moment they're first fetched. This isn't perfect, though, as it may give inaccurate cost calculations. Unassociated IP addresses will only start counting towards the cost after Komiser fetches them, which could lead to underestimating the actual costs.

Can you think of any better way? 
Cheers

### CODE REVIEW

**Review by eneskaya — APPROVED** [review]

File: providers/aws/ec2/elastic-ips.go [hunk 1]
```diff
@@ -43,13 +43,23 @@ func ElasticIps(ctx context.Context, client ProviderClient) ([]Resource, error)
 
 			resourceArn := fmt.Sprintf("arn:aws:ec2:%s:%s:elastic-ip/%s", client.AWSClient.Region, *accountId, *elasticIps.AllocationId)
 
+			fetchedAt := time.Now()
+			hoursSinceFetched := hoursSince(fetchedAt)
```
Comment (eneskaya) [inline providers/aws/ec2/elastic-ips.go:47]: Wait, I approved but now I just saw, that this will be always `0` hours, hence 0$ monthly cost

**Review by mlabouardy — CHANGES_REQUESTED** [review]
You can get the creation date of the IP by leveraging AWS Config. 
This command should return the creation date (under `configurationItemCaptureTime` attribute):
` aws configservice batch-get-resource-config --resource-keys resourceType=AWS::EC2::EIP,resourceId=eipalloc-ID_HERE`

Screenshot 2023-04-05 at 10 29 48

**Comment by jakepage91** [comment]
Hey there @eneskaya and @mlabouardy just to let you know that I made some updates (thanks for the help Moh!)

**Review by mlabouardy — APPROVED** [review]
@jakepage91 I've added missing AWS permission to IAM policy file as well handled when the permission is missing otherwise the resource won't be collected

**Pull request merged**